eng/Versions.props

```diff
@@ -35,6 +35,12 @@
     <SystemSecurityPermissionsPackageVersion>7.0.0-alpha.1.21516.3</SystemSecurityPermissionsPackageVersion>
     <SystemWindowsExtensionsPackageVersion>7.0.0-alpha.1.21516.3</SystemWindowsExtensionsPackageVersion>
   </PropertyGroup>
+
+  <!-- Docs / Intellisense -->
+  <PropertyGroup>
+    <MicrosoftPrivateIntellisenseVersion>6.0.0-preview-20211015.1</MicrosoftPrivateIntellisenseVersion>
```

Comment:
You'd want to pin it to 6.0.0-preview-20210916.1 for now, as the version you use doesn't contain Windows Desktop xmls.